frontend/src/components/validation/bootstrap-form-renderer.js

```diff
@@ -0,0 +1,64 @@
+import {
+    ValidationRenderer,
+    RenderInstruction,
+    ValidateResult
+} from 'aurelia-validation';
+
+export class BootstrapFormRenderer {
+    render(instruction: RenderInstruction) {
+        for (let { result, elements } of instruction.unrender) {
+            for (let element of elements) {
+                this.remove(element, result);
+            }
```

Comment:
maybe you can extract this look into a function and provide 
`instruction.unrender` & `instruction.render` as parameter 
and `this.remove` & `this.add` as callbacks to avoid code duplication